src/constants/screens.ts

```diff
@@ -13,9 +13,10 @@ type MainTabsParamsList = {
 }
 
 type HomepageStackParamList = {
-    Homepage: undefined
```

Comment:
For this you can do: 
`Homepage?: {searchValue: string}`

Reply:
is this alright as well? 
Homepage: { searchValue: string } | undefined